fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/RuntimeFilter.java

```diff
@@ -30,21 +30,37 @@ public class RuntimeFilter {
     private final RuntimeFilterId id;
     private final TRuntimeFilterType type;
     private final Expression srcSlot;
+    //bitmap filter support target expression like  k1+1, abs(k1)
+    //targetExpression is an expression on targetSlot, in which there is only one non-const slot
+    private Expression targetExpression;
     private Slot targetSlot;
     private final int exprOrder;
-    private PhysicalHashJoin builderNode;
+    private AbstractPhysicalJoin builderNode;
+
+    private boolean bitmapFilterNotIn;
```

Comment:
the attr name could be more generation such as `reverse` or `anti`

Reply:
as we discussed, BE only support semi, not anti.